src/miral/basic_window_manager.cpp

```diff
@@ -1280,7 +1280,8 @@ void miral::BasicWindowManager::place_attached_to_zone(
     }
 
     default:
-        fatal_error("BasicWindowManager::place_maximized() called for window not in a maximized or attached state");
+        fatal_error("BasicWindowManager::place_attached_to_zone() called for window not in a maximized or attached state\n"
+                    "Please report here: https://github.com/MirServer/mir/issues/new");
```

Comment:
This is code noise here. If anything, a link to report issues should go into `fatal_error()`. That's not related to this PR though.